docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+
+To initialize a local Git repository and run the command:
+
+```shell
+git init
+```
+
+Now add your code and commit it to the repository:
+
+```shell
+git add .
+git commit -m "Initial commit"
+```
+
+**2. Add a Procfile**
+
+Every Heroku app uses a special file called `Procfile` that tells Heroku how to start your app. A Bolt Slack app will be started as a web server with a public web address.
+
+Create a new file called `Procfile` (without any extension) in your app's root directory with the following content:
+
+```yaml
+web: node app.js
+```
+
+Once you've saved the file, let's commit it to your Git repository:
+
+```shell
+git add Procfile
+git commit -m "Add Procfile"
+```
+
+**3. Extra steps for existing apps**
+
+Are you following this guide with an existing app that you created? If so, please review [preparing a codebase for Heroku](https://devcenter.heroku.com/articles/preparing-a-codebase-for-heroku-deployment#4-listen-on-the-correct-port) to listen on the correct port and think about data storage provisioning.
+
+---
+
+### Setup the Heroku tools
+
+Now we can set up the Heroku tools on your local machine. These tools will help you manage, deploy, and debug your app on Heroku's platform.
+
+**1. Install the Heroku CLI**
+
+The Heroku tools are available as a Command Line Interface (CLI). Go ahead and [install the Heroku CLI for macOS, Windows, or Linux](https://devcenter.heroku.com/articles/getting-started-with-nodejs#set-up). On macOS, you can run the command:
+
+```shell
+brew install heroku/brew/heroku
+```
+
+Once the install is complete, we can test the Heroku CLI by displaying all of the wonderful commands available to you:
+
+```shell
+heroku help
+```
+
+> 💡 If the `heroku` command is not found, you can refresh your path by opening a new terminal session/tab.
+
+**2. Log into the Heroku CLI**
+
+The Heroku CLI connects your local machine with your Heroku account. [Sign up for a free Heroku account](https://heroku.com) and then log into the Heroku CLI with the following command:
+
+```shell
+heroku login
+```
+> 💡 If you're behind a firewall, you may need to [set the proxy environment variables](https://devcenter.heroku.com/articles/using-the-cli#using-an-http-proxy) for the Heroku CLI.
+
+**3. Confirm you're logged into the Heroku CLI**
+
+Let's check that you're logged in by displaying the account currently connected to your Heroku CLI:
+
+```shell
+heroku auth:whoami
+```
+
+You're all setup with the Heroku tools! Now let's move onto the exciting step of creating an app on Heroku.
+
+---
+
+### Create an app on Heroku
+
+It’s time to [create a Heroku app](https://devcenter.heroku.com/articles/creating-apps) using the tools that you just installed. When you create an app, you can choose a unique name or have it randomly generated.
+
+> 💡 You can [rename a Heroku app](https://devcenter.heroku.com/articles/renaming-apps) at any time, but you may need to update your Git remote and app URL.
```

Comment:
I think you can delete this

Reply:
@shaydewael if possible, I'd like to keep this section. In the Getting Started guide, I really appreciated the note that we can rename our Slack app after creating it because I was immediately struck with _naming paralysis_. I felt this call out would do the same to ease developers fears about naming their Heroku app.

If you strongly feel we can remove this section, just let me know and I'll delete it!